documentation/api/io.strimzi.api.kafka.model.template.ContainerTemplate.adoc

```diff
@@ -0,0 +1,25 @@
+You can set custom security context and environment variables for a container.
```

Comment:
Do we want to give examples/scenarios of when we set both?

Reply:
I don't think so. It is variety of weird usecases which are not really common. So I do not think it makes much sense.